server/routes/projects.js

```diff
@@ -1,14 +1,39 @@
-import dbConnection from "../services/dbConnection.js";
-
-const connection = new dbConnection();
-
 export default app => {
   app.get("/projects", (req, res) => {
-    connection
-      .query("SELECT * FROM Projects")
-      .then(result => {
-        res.send(result);
-      })
-      .then(() => connection.close());
+    res.send("all projects");
+  });
+
+  app.post("/projects", (req, res) => {
```

Comment:
I think I've seen this before :) You probably want to remove the duplicated code from `dist/routes/`

Reply:
Sorry, it's my mistake. `dist` directory shouldn't be on the remote repository. I needed it to use ES6 in the server project.